WallsAndHoles/tiletemplatepropertymanager.cpp

```diff
@@ -44,7 +44,7 @@ QVector<QVector<QVariant>> TileTemplatePropertyManager::properties()
         {"Name",       mTileTemplate->name(),         true              },
         {"Color",      mTileTemplate->color(),        true              },
         {"Height",     mTileTemplate->height(),       true,  -1000, 1000},
-        {"Thickness",  mTileTemplate->thickness(),    false,   0.1,    1},
+        {"Thickness",  mTileTemplate->thickness(),    true,    0.1,    1},
         {"X Position", mTileTemplate->position().x(), false, -0.49, 0.49},
```

Comment:
Can't position also be allowed to be modified now?

Reply:
Yep.